runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -391,7 +391,7 @@ public String execModule(String fileName) {
 			ProcessBuilder builder = new ProcessBuilder(goPath, "run", modulePath,
 					inputPath);
 			builder.environment().put("GOPATH",
-					runtimePath + File.pathSeparator + tmpdir);
+					builder.environment().get("GOPATH") + ":" + runtimePath + File.pathSeparator + tmpdir);
```

Comment:
What happens if this environment variable is not defined? Is it possible we could throw a more verbose message or try something else?